### DISCUSSION TITLE
Understanding charging level matter

### DISCUSSION THREAD

**Discussion by chdslv** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1364] [General]
I installed AlDente yesterday on my 2018 15" MacBook Pro. The battery is quite decent, holding steady at 88%. I've been using the optimised charging mode in macOS since then. That feature worked rather well in Sonoma and earlier versions, maintaining the charge at around 80% for weeks—actually, it was 77% according to CoconutBattery. However, since the Sequoia update, that mode has gone a bit awry, often keeping the charge level at 100%.

In AlDente, I set the charge limit to 74% and disabled the optimised battery charging mode, then rebooted the MacBook Pro. At that point, I had it unplugged, with the battery level below 60%. Once I plugged it back in, the battery level rose to 76% (as per Apple) and 72.4% (according to CoconutBattery). I’m not overly concerned about the 2% discrepancy. I then topped the battery up to 90% (Apple) and clicked on discharge. The AlDente/MacBook Pro began using the battery, and when it reached 76% (Apple), it stopped, and AlDente took over. It maintained that level all night, even while in sleep mode. It’s still at 76%, and the battery isn’t charging, with the temperature remaining at 26.7°C.

Now, with all that out of the way, my question is: should I wait two weeks before disabling AlDente and using the MacBook Pro on battery for a day? I used to do that about once a month with macOS’s own optimised charging mode. I’ve noticed that AlDente doesn’t seem to use a bit of battery and then charge it back to the set level. Perhaps that feature is available in the pro version with its sailing mode. This brings me to another question: keeping the battery level stable for two weeks might not be feasible, as the battery will naturally drain over time. Will the standard version of AlDente charge it back to the planned limit—in my case, 74%—within those two weeks?

**Comment by MatthiasKerbl** [comment]
Hi @chdslv,

> Will the standard version of AlDente charge it back to the planned limit—in my case, 74%—within those two weeks?

Yes, AlDente Free will always keep your battery at the charge limit. This means AlDente will microcharge the battery every time it falls below the charge limit.

Best,
Matthias

**Reply by chdslv** [reply]
Thanks for the quick reply. I'll keep an eye on this for few days check if it does that. Why does it stay at 76% instead of 74%? 
![Screenshot 2024-11-05 at 10 07 58](https://github.com/user-attachments/assets/e238ad0b-24fb-4daa-b771-47dcccea1930)

**Reply by MatthiasKerbl** [reply]
Due to a different technical pathway used to control the charging behavior on Intel MacBooks compared to Apple Silicon MacBooks, the charge limit might be slightly more inaccurate.

Best,
Matthias

**Reply by chdslv** [reply]
Will report after 2 weeks. 😊

**Reply by chdslv** [reply]
By the way, should I disable AiDente, before upgrading to a new OS release?

**Reply by MatthiasKerbl** [reply]
No, in general, this is not necessary. If you run into issues with AlDente after upgrading macOS, you can fix it again by unplugging and restarting the MacBook.

Best,
Matthias

**Reply by chdslv** [reply]
It's quite interesting how your AiDente functions following the OS upgrade. The battery level only reached 80% (Apple), and over the past two days, it's gradually decreased to 77% (Apple). AiDente micro-charges, keeping the battery temperature low. I believe it does its job well, not as erratic as Apple's own script. 👍

**Comment by chdslv** [comment]
Thanks! 👍
